src/chttpd/src/chttpd_db.erl

```diff
@@ -334,7 +334,8 @@ handle_design_req(#httpd{
         path_parts=[_DbName, _Design, Name, <<"_",_/binary>> = Action | _Rest]
     }=Req, Db) ->
     DbName = fabric2_db:name(Db),
-    case ddoc_cache:open(DbName, <<"_design/", Name/binary>>) of
+%%    case ddoc_cache:open(DbName, <<"_design/", Name/binary>>) of
```

Comment:
Don't add commented code if possible

Reply:
I'm happy to remove it if we won't implement a ddoc_cache soon?

Reply:
Up to you, it's just a suggestion. If we'll have a ddoc_cache soon let's keep it.

Reply:
I'm usually a pretty big stickler on this but left it as a note that we disabled the ddoc cache here. I'm fine removing the code but we should probably keep a note that it used be to cached here so we can revisit after we update the ddoc cache implementation.